sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/TokenResult.java

```diff
@@ -123,6 +130,17 @@ public TokenResult withExpirationTime(Date expiresOn) {
         return this;
     }
 
+    /**
+     * Set the account id.
+     *
+     * @param accountId Account id value to be set.
+     * @return The TokenResult object itself.
+     */
+    public TokenResult withAccountId(String accountId) {
```

Comment:
The "with" convention is only used in apple SDK.

```suggestion
    public TokenResult setAccountId(String accountId) {
```

Reply:
sure, will fix it latter :) overall files, currently the naming is in all models

Reply:
Let's get that merged, then change everywhere when noone is modifying it.